garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -96,6 +106,14 @@ local function ColorForPlayer(ply)
    return namecolor.default
 end
 
+local function RowColorForPlayer(ply)
+   if IsValid(ply) then
+      return hook.Call("TTTScoreboardRowColorForPlayer", GAMEMODE, ply)
+   end
+
+   return Color(0, 0, 0, 0)
+end
+
 function PANEL:Paint()

```

Comment:
`width` and `height` parameters to use on [line 129](https://github.com/garrynewman/garrysmod/pull/1250#discussion-diff-81411277R129)